components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -93,8 +93,38 @@ class TabsUseCases(
         }
     }
 
+    class RemoveAllTabsUseCase internal constructor(
+        private val sessionManager: SessionManager
+    ) {
+
+        fun invoke() {
+            sessionManager.removeAll()
+        }
+    }
+
+    class RemoveAllTabsOfTypeUseCase internal constructor(
+        private val sessionManager: SessionManager
+    ) {
+        /**
+         * @param private pass true if only private tabs should be removed otherwise normal tabs will be removed
+         */
+
+        fun invoke(private: Boolean) {
+            sessionManager.all.filter { it.private == private }.forEach {
+
+                sessionManager.remove(it)
+            }
+        }
+        }
+
     val selectTab: SelectTabUseCase by lazy { SelectTabUseCase(sessionManager) }
     val removeTab: RemoveTabUseCase by lazy { RemoveTabUseCase(sessionManager) }
     val addTab: AddNewTabUseCase by lazy { AddNewTabUseCase(sessionManager) }
     val addPrivateTab: AddNewPrivateTabUseCase by lazy { AddNewPrivateTabUseCase(sessionManager) }
+    val removeAllTabs: RemoveAllTabsUseCase by lazy { RemoveAllTabsUseCase(sessionManager) }
+    val removeAllTabsOfTypeUseCase: RemoveAllTabsOfTypeUseCase by lazy {
```

Comment:
We don't need to add 'UseCase' to the attribute binding. :)

`removeAllTabsOfTypeUseCase` -> `removeAllTabsOfType`

Reply:
ok.sure will do change